### ISSUE TITLE
Can we get a new release?

### ISSUE THREAD

**Issue by frebib** [matrix-org/matrix-synapse-ldap3#185]
In particular, I'm wanting https://github.com/matrix-org/matrix-synapse-ldap3/commit/c65e8cbd27a5cd935ce12e7c4b92143cdf795c86

Currently I have to uninstall/reinstall this from git every time I (re) start my synapse container, which is certainly less than ideal. It'd be nice if the version that ships with Synapse is up-to-date

**Comment by DMRobertson** [comment]
Yes we can: https://github.com/matrix-org/matrix-synapse-ldap3/releases/tag/v0.3.0

> It'd be nice if the version that ships with Synapse is up-to-date

We should be able to arrange this too.

**Comment by frebib** [comment]
Thanks!

**Comment by DMRobertson** [comment]
https://github.com/matrix-org/matrix-synapse-ldap3/issues/185 should land when in the next-cut Synapse release. I think that's 1.96, due to be cut on 31st Oct and released 7th November.
